client/components/forms/form-text-input/index.jsx

```diff
@@ -40,7 +43,7 @@ export default class FormTextInput extends PureComponent {
 			<input
 				type="text"
 				{ ...props }
-				ref="textField"
+				ref={ inputRef || 'textField' }
```

Comment:
I'd guess this could get messy if the parent component assigned both `ref` and `inputRef`, then attempted to call `this.refs.foo.focus()`, yeah? Edge case certainly.